src/views/LoginSuccess.vue

```diff
@@ -176,6 +179,13 @@ export default class LoginSuccess extends Vue {
             await new Promise((resolve) => { this.resolve = resolve; });
         }
 
+        if (this.request.kind === RequestType.CHANGE_PASSWORD) {
```

Comment:
Might want to add a comment how we can end up from a `CHANGE_PASSWORD` request in `LoginSuccess`.